app/models/player.rb

```diff
@@ -1,4 +1,5 @@
 class Player < ApplicationRecord
   belongs_to :tournament
   has_many :matchups
+  validates :name, :rating, :division, :crosstables_id, :tournament_id, presence: true
```

Comment:
If we're allowing functionality to manually add users who aren't currently on cross-tables (e.g. a first time tournament player who joined late), we shouldn't require that field